examples/guestbook/README.md

```diff
@@ -392,7 +392,7 @@ spec:
     spec:
       containers:
       - name: php-redis
-        image: kubernetes/example-guestbook-php-redis:v2
+        image: kubernetes/example-guestbook-php-redis:v3
```

Comment:
As we are on this, could you change it to gcr.io/google_containers/example-guestbook-php-redis:v3?